### DISCUSSION TITLE
RSSI Heatmap from multiple APs

### DISCUSSION THREAD

**Discussion by laurenchristyt** [NVlabs/sionna#344] [Q&A] [answered]
Hello, I am a student and I'm very interested in learning about Sionna and how to use it in a Wi-Fi management system. Specifically, I aim to create an RSSI Heatmap by collecting data from multiple access points within an area. Is it possible to achieve this using Sionna?

**Comment by faycalaa** [comment, marked as answer]
Hi,

Sionna can [generate pathloss heatmaps for multiple transmitters](https://nvlabs.github.io/sionna/api/rt.html#coverage-map).
You would then need to consider the transmit power to calculate the RSSI.

Hope this helps.
